docs/workloads/azure/data/components/etl_pipelines/ingest_data_azure.md

```diff
@@ -14,7 +14,7 @@ keywords:
 The solution contains a sample Azure Data Factory pipeline for ingesting data from a sample data
 source (Azure SQL) and loading data into the data lake 'landing' zone.
 
-Link to the pipeline: [stacks-azure-data/ingest](https://github.com/amido/stacks-azure-data/tree/main/ingest).
+Link to the pipeline: [stacks-azure-data/de_workloads/ingest](https://github.com/amido/stacks-azure-data/tree/main/de_workloads/ingest/Ingest_AzureSql_Example).
```

Comment:
Looks good, I just suggest to apply these changes to Amido repo first to make sure everything builds.